alchemist/alchemist-swingui/src/main/java/it/unibo/alchemist/boundary/wormhole/implementation/adapter/NodeViewType.java

```diff
@@ -55,6 +55,7 @@ public double getHeight() {
     /**
      * @inheritDocs
      */
+    @SuppressWarnings("checkstyle:MagicNumber")
     @Override
     public boolean equals(final Object o) {
```

Comment:
Why is equals overridden? Does this object end up as key in some map?

Reply:
Not as far as I could find.

Reply:
Remove equals and hashcode, let's inherit from Object